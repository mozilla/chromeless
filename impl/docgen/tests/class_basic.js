/**
 * A test of basic class parsing
 */

/**
 * @class URL
 * A class which parses a url and exposes its various
 * components separately.
 */

/** @endclass */
