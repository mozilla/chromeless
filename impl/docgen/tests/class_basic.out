{
  "classes": [
    {
      "desc": "A class which parses a url and exposes its various\ncomponents separately.", 
      "name": "URL"
    }
  ], 
  "desc": "A test of basic class parsing", 
  "filename": "tests/class_basic.js", 
  "module": "class_basic"
}
